client/feed.js

```diff
@@ -0,0 +1,55 @@
+import React, { Component } from 'react'
+import ReactDOM from 'react-dom'
+
+export class Feed extends Component {
+  constructor() {
+    super()
+    this.state = {}
+    this.renderFeedStory = this.renderFeedStory.bind(this)
+  }
+  componentDidMount() {
+    fetch('http://127.0.0.1:3000/feed')
+      .then(res => res.json())
+      .then(data => {
+        this.setState({ channel: data.channel, feedStories: data.stories })
+      })
+      .catch(err => {
```

Comment:
It's a good idea to render and error message instead of hiding the problem in the console.